src/modules/components/proposal/proposalActions/actions/mocks/proposalActionTokenMint.ts

```diff
@@ -0,0 +1,112 @@
+import { type IProposalActionTokenMint } from '../../proposalActionsTypes/proposalActionTokenMint';
+
+export const tokenMintActionsMock: IProposalActionTokenMint[] = [
```

Comment:
We usually use generator functions to generate mock data needed for tests. In this way, if something changes on the interface that does not affect the tests, we only need to update the generator. You'll find more examples on the app-next repository